packages/vector_graphics/lib/src/vector_graphics.dart

```diff
@@ -510,11 +517,24 @@ class _VectorGraphicWidgetState extends State<VectorGraphic> {
         _stackTrace ?? StackTrace.empty,
       );
     } else {
-      child = widget.placeholderBuilder?.call(context) ??
-          SizedBox(
-            width: widget.width,
-            height: widget.height,
-          );
+      child = widget.placeholderBuilder != null
```

Comment:
Instead of the double nested ternary please right a helper function and/or some if statements.